tests/installation/partitioning/warning/bios_boot_small_for_bootloader.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify Warning Dialog when BIOS boot partition has too small
+# size to install bootloader.
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+use scheduler 'get_test_suite_data';
+use Test::Assert ':all';
+
+my $partitioner;
+
+sub run {
+    my $test_data = get_test_suite_data();
+    my $disk      = $test_data->{disks}[0];
+    $partitioner = $testapi::distri->get_expert_partitioner();
+
+    $partitioner->run_expert_partitioner();
+    foreach my $partition (@{$disk->{partitions}->{bios_boot_small_for_bootloader}}) {
+        $partitioner->add_partition_on_gpt_disk({
+                disk      => $disk->{name},
+                partition => $partition
+        });
+    }
+    $partitioner->accept_changes();
```

Comment:
looks to me that from here to above is just one module that we could repeate in the schedule multiple times. but that is for later when we can see all the warnings to think about it.

Reply:
I agree! The flow is similar and sometimes even same. The only issue is that there are 3 types of warnings may happen in the end: 
 - Error popup with only "Ok" button;
 - Warning with YLabel containing the warning message (and "Yes", "No" buttons);
 - Warning with YRichText containing the warning message (and "Yes", "No" buttons).

Due to that, it is not so easy from the logic prospective to mix them in one and convert to pure data-driven test.

Reply:
yes, I only meant the partitioning module.